feature-pack/src/main/resources/modules/system/layers/base/org/wildfly/extension/microprofile/fault-tolerance-smallrye/main/module.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~ Copyright 2019, Red Hat, Inc., and individual contributors
+  ~ as indicated by the @author tags. See the copyright.txt file in the
+  ~ distribution for a full listing of individual contributors.
+  ~
+  ~ This is free software; you can redistribute it and/or modify it
+  ~ under the terms of the GNU Lesser General Public License as
+  ~ published by the Free Software Foundation; either version 2.1 of
+  ~ the License, or (at your option) any later version.
+  ~
+  ~ This software is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  ~ Lesser General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Lesser General Public
+  ~ License along with this software; if not, write to the Free
+  ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+  ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+  -->
+
+<module xmlns="urn:jboss:module:1.9" name="org.wildfly.extension.microprofile.fault-tolerance-smallrye">
+    <properties>
+        <property name="jboss.api" value="private"/>
+    </properties>
+
+    <resources>
+        <artifact name="${org.wildfly:wildfly-microprofile-fault-tolerance-smallrye-extension}"/>
+    </resources>
+
+    <dependencies>
+        <module name="com.netflix.archaius.core"/>
+        <module name="io.smallrye.fault-tolerance" export="true"/>
+        <module name="javax.annotation.api"/>
+        <module name="javax.enterprise.api"/>
+        <module name="org.apache.commons.configuration"/>
+        <module name="org.eclipse.microprofile.config.api"/>
+        <module name="org.eclipse.microprofile.fault-tolerance.api"/>
+        <module name="org.jboss.as.clustering.common"/>
+        <module name="org.jboss.as.controller"/>
+        <module name="org.jboss.as.server"/>
+        <module name="org.jboss.as.weld.common"/>
+        <module name="org.jboss.logging"/>
+        <module name="org.jboss.modules"/>
+        <module name="org.jboss.msc"/>
+        <module name="org.jboss.staxmapper"/>
+        <module name="org.jboss.weld.api"/>
+        <module name="org.jboss.weld.core"/>
+        <module name="org.jboss.weld.spi"/>
+        <module name="org.wildfly.microprofile.fault-tolerance-smallrye.cdi"/>
+    </dependencies>
```

Comment:
The weld ones are needed to have proper proxy generation. Otherwise you get a warning message that they will be created in the deployment classloader which is problematic if the proxied class has package protecte methods and things like that.

I have added similar eclipse and smallrye dependencies for my MP specs and they are essentially needed to have those on the classpath so it scans for CDI beans to wire up all the MP magic

Reply:
Cool!, good to know it. I see some of them are being already exported by `io.smallrye.fault-tolerance`, I'm still with doubts about the intention if they are exported why add them here, maybe the export is unnecessary. My point is to not add more than needed since it helps Galleon in reducing the size when we have custom provisioning. 

So, then we can reduce the list to only `org.jboss.msc` and `javax.annotation.api`

Reply:
I have removed the unneeded ones now. Thanks!